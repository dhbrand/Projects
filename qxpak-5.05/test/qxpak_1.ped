1 0 0 1 2
2 0 0 2 2
3 1 2 1 1 
4 1 2 2 1
5 1 2 1 1 
6 1 2 1 1
7 1 2 1 1
8 1 2 1 1
9 3 2  1 1
10 3 4 1 1
11 0 0 1 1 
12 0 0 2 1
13 11 12 1 1
14 11 12  1 1
15 11 12 1 1
16 11 12 1 1
17 11 12 1 1
18 11 12 1 1
19 13 12 1 1
20 13 14 1 1
