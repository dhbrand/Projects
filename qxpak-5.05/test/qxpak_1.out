  
              ***************
              *** QXPAK.5 ***
              ***************
               MPE / IM 2010

 Qxpak v. 5.05  
 Date yy/mm/dd 2018/05/16
 Time hh/mm/ss   14/01/44
 Statistical method:        ML                  
 Parameter file:            qxpak_1.par.tmp                         
 Data file:                 qxpak_1.dat                             
 Pedigree file              qxpak_1.ped                             
 Marker file                qxpak_1.mkr                             
 Number of traits:                     1
 Number of inds:                      20
 Number of chrs:                       1
 Number of markers / chr               4
 Number of breeds:                     2
 Marker positions           chr1    0.0    5.0   10.0   15.0
 Number of QTL:                        1
 Number of Epistatic QTL:              0
 Number of effects:                    5
 Number of effects w/o QTL:            3
 Value of missing trait:               0
 QTL scan step (cM):         1.00
 Number of MCMC iters:              1000

SCANNED QTL POSITIONS
     QTL    Type   Imprint   Segment  chr1          
      qtl1  fix_ad     -        -     0.00  15.00

 EFFECTS
 Name          # levels      Type      Randomtype     File
                                      (Nested cov)
 qtl1_a               1   covariable                                 
 qtl1_d               1   covariable                                 
 mean                 1   cross-clas                                 
 age                  1   covariable                                 
 u0                  20   cross-clas   add. animal    qxpak_1.ped    

 INITIAL variances
 Residual
 t1       13.0714
 u0                                      
 t1       10.4571

   1    1.00          -28.794204    21.940

 MODEL           0  specifications
 Trait    Position   Effects
                     qtl1_a    qtl1_d    mean      age       u0        
 t1           5      -         -         +         +         +        

 STATISTICS
 Effect/trait      N    Mean     sd      Min    Max
  mean              7   1.000   0.000   1.000   1.000
  age               7   3.353   0.376   2.600   3.800
  u0                7   7.286   2.373   4.000  11.000
  t1                7  11.143   4.734   6.000  19.000

 Maximum likelihood: 
 2logL = constant +           -28.794204

 RESIDUAL variances
          t1        
 t1       21.9398

 EFFECT variances: u0                                      
          t1        
 t1        1.0770

 EFFECT: h2 in diagonal / correlations off-diagonal u0                                      
          t1        
 t1        0.0468

 SOLUTIONS
 trait/effect level   solution    std. error
  1   3        1        5.2083       16.2538
  1   4        1        1.7621        4.8332
  1   5        1        0.0422        1.0360
  1   5        2        0.0422        1.0360
  1   5        3       -0.0001        1.0348
  1   5        4        0.1158        1.0291
  1   5        5       -0.1014        1.0344
  1   5        6        0.2168        1.0330
  1   5        7       -0.0573        1.0287
  1   5        8        0.1216        1.0270
  1   5        9        0.0210        1.1584
  1   5       10       -0.0268        1.1480
  1   5       11       -0.0844        1.0308
  1   5       12        0.0000        1.0378
  1   5       13       -0.0422        1.0360
  1   5       14       -0.0422        1.0360
  1   5       15       -0.0422        1.0360
  1   5       16       -0.0422        1.0360
  1   5       17       -0.0422        1.0360
  1   5       18       -0.0422        1.0360
  1   5       19       -0.0211        1.1599
  1   5       20       -0.0422        1.1587

 QTL SCAN
  Positions  df          2logLRatio  Pvalue  QTL_estimates  Residual_Variances
   1    1.00    1         1         0.510289  0.475E+00    -3.119     0.000    20.397
   2    2.00    2         1         0.510289  0.475E+00    -3.119     0.000    20.397
   3    3.00    3         1         0.510289  0.475E+00    -3.119     0.000    20.397
   4    4.00    4         1         0.510289  0.475E+00    -3.119     0.000    20.397
   5    5.00    5         1         0.510289  0.475E+00    -3.119     0.000    20.397
   6    6.00    6         1         0.510289  0.475E+00    -3.119     0.000    20.397
   7    7.00    7         1         0.510289  0.475E+00    -3.119     0.000    20.397
   8    8.00    8         1         0.510289  0.475E+00    -3.119     0.000    20.397
   9    9.00    9         1         0.510289  0.475E+00    -3.119     0.000    20.397
  10   10.00   10         1         0.510289  0.475E+00    -3.119     0.000    20.397
  11   11.00   11         1         0.510289  0.475E+00    -3.119     0.000    20.397
  12   12.00   12         1         0.510289  0.475E+00    -3.119     0.000    20.397
  13   13.00   13         1         0.510289  0.475E+00    -3.119     0.000    20.397
  14   14.00   14         1         0.510289  0.475E+00    -3.119     0.000    20.397
  15   15.00   15         1         0.510289  0.475E+00    -3.119     0.000    20.397

 MODEL           1  specifications
 Trait    Position   Effects
                     qtl1_a    qtl1_d    mean      age       u0        
 t1           5      +         +         +         +         +        

 Likelihood ratio: 
  LR =   0.51028915257104757     
 Estimated d.f.             1
 Nominal P-value  0.475E+00
 For completeness, nominal P-values for other d.f.
 d.f.  2 P-value  0.775E+00

 QTL positions for LR max      1.00

 RESIDUAL variances
          t1        
 t1       20.3969

 EFFECT variances: u0                                      
          t1        
 t1        1.0286

 EFFECT: h2 in diagonal / correlations off-diagonal u0                                      
          t1        
 t1        0.0480

 SOLUTIONS
 trait/effect level   solution    std. error
  1   1        1       -3.1192        4.3202
  1   2        1        0.0000        0.0000
  1   3        1       18.7142       24.4058
  1   4        1       -2.9212        7.9876
  1   5        1       -0.0000        1.0142
  1   5        2       -0.0000        1.0142
  1   5        3       -0.0434        1.0129
  1   5        4        0.0593        1.0088
  1   5        5       -0.1174        1.0110
  1   5        6        0.1978        1.0100
  1   5        7       -0.1266        1.0094
  1   5        8        0.0303        1.0118
  1   5        9       -0.0217        1.1337
  1   5       10       -0.0789        1.1238
  1   5       11       -0.0000        1.0142
  1   5       12        0.0000        1.0142
  1   5       13       -0.0000        1.0142
  1   5       14       -0.0000        1.0142
  1   5       15       -0.0000        1.0142
  1   5       16       -0.0000        1.0142
  1   5       17       -0.0000        1.0142
  1   5       18       -0.0000        1.0142
  1   5       19       -0.0000        1.1339
  1   5       20       -0.0000        1.1339
 Total CPU time:           0 h           0 m           6 s
